cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,24 @@
-describe('Sign Up page', () => {
-  it('should provide an ability to register new account', () => {
+import { generateUser } from '../support/generate';
 
+describe('User Registration and Login Tests', () => {
```

Comment:
Why do we need describe inside describe? Better to use context() inside describe, but in this cases it is okay to have one describe